lib/scenic/adapters/postgres.rb

```diff
@@ -81,6 +81,29 @@ def update_view(name, sql_definition)
         create_view(name, sql_definition)
       end
 
+      # Updates a view in the database using `CREATE OR REPLACE VIEW`.
+      #
+      # This results in a `CREATE OR REPLACE VIEW`. Most of the time the
+      # explicitness of the two step process used in {#update_view} is preferred to
+      # `CREATE OR REPLACE VIEW` because the former ensures that the view you are trying
+      # to update did, in fact, already exist. Additionally, `CREATE OR REPLACE
```

Comment:
We *could* verify this as well if we want to avoid calling this method to replace a non-existent view. What do you think?